.commitlintrc.cjs

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  extends: ['@commitlint/config-conventional'],
+  rules: {
+    "type-enum": [2, "always", ['feat', 'fix', 'docs', 'style', 'refactor', 'test', 'revert', 'build', 'chore', 'ci']]
+  }
+}
```

Comment:
@commitlint/config-conventional enumerates types, so you'll get the friendly "must be one of" error without adding the type-enum here

Reply:
(late comment — opened the link from slack, but didn't look at it until now:)

Reply:
I included this because I wanted to customize/extend the allowed types :)

Reply:
gotcha cool. so (just chatting now) is the opinion that "perf" should instead be rolled into … maybe "feat"?

Reply:
oh oh I see https://commitlint.js.org/#/reference-rules?id=type-enum lies. the fact that your last three come after revert makes me think you may have been fooled by it. https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional

Reply:
OOOO yep, this got me. I think I can just remove it unless I have any additional types I want to add. Thanks!